requirements-development.txt

```diff
@@ -10,6 +10,7 @@ django-allauth==0.37.1
 django-countries==5.3.2
 django-rest-auth==0.9.3
 djangorestframework==3.8.2
+factory-boy==2.11.1
```

Comment:
If it is is normal requirements we don't have to add it also to requirements-develpment I think :)

Reply:
Done :+1: